parsers/verilog.c

```diff
@@ -132,6 +133,7 @@ typedef struct sTokenInfo {
 	bool                classScope;    /* Context is local to the current sub-context */
 	bool				parameter;	   /* parameter which can be overridden */
 	bool				hasParamList;  /* module definition has a parameter port list */
+	bool		    virtual;       /* has virtual */
```

Comment:
I don't care about whitespaces in code that I don't maintain directly.
However, you can care:-).

![image](https://github.com/universal-ctags/ctags/assets/77077/777d54a6-6b28-4678-a48c-6f34c8b57db0)

If you use Emacs, M-x whitespace-mode helps you see how the sequence of whitespace is made.

Reply:
@masatake san,

> However, you can care:-).

Sorry.  I have been caring about it.

The indentation rules for this project are different from those I usually use.
I worked on a new PC that had not yet been configured for this project. I wrongly indented the code by hand and committed it.

I have corrected the original commit.

I was also concerned about the inconsistent use of spaces in the indentation. I took this opportunity to add a commit that fixes that as well.

Could you review it again?

Reply:
About the style of spacing(?), I don't have a strong preference.
If the file's maintainer feels comfortable, that's good.


As a reference, we have .editorconfig globally used in our code base.
If you have not introduced editorconfig on your new PC, this will be a good time to introduce it.